server/db/Models/Cart.js

```diff
@@ -0,0 +1,26 @@
+const Sequelize = require('sequelize');
+
+const {
+  UUID, UUIDV4, STRING,
+} = Sequelize;
+const { db } = require('../db');
+
+const Cart = db.define('Cart', {
+  id: {
+    type: UUID,
+    defaultValue: UUIDV4,
+    primaryKey: true,
+  },
+  items: {
+    type: STRING,
+    allowNull: false,
+    get() {
+      return this.getDataValue('items').split(';');
+    },
+    set(val) {
+      this.setDataValue('items', val.join(';'));
+    },
```

Comment:
I like this idea... it may require a little extra work on the front end displaying/handling the quantities but I think that stuff is fun to figure out.  Passing around a string amongst these models feels manageable